std/bitmanip.d

```diff
@@ -3994,17 +3994,17 @@ if (canSwapEndianness!T && isOutputRange!(R, ubyte))
 {
     import std.array;
     import std.format : format;
-    import std.meta;
-    foreach (endianness; AliasSeq!(Endian.bigEndian, Endian.littleEndian))
-    {
+    import std.meta : AliasSeq;
+    static foreach (endianness; [Endian.bigEndian, Endian.littleEndian])
```

Comment:
Why the change from `AliasSeq` to an array?

Reply:
Because `static foreach` can unroll arrays and thus there's no need for `AliasSeq` anymore.

Reply:
Yes but why only this one specifically? I know most of these loops are over lists of types, but there's at least one other one that loops over a list of values.

Reply:
Then I overlooked the other case. This PR catches only 95% of the `static foreach`'s.
Anyways, we can easily make more improvements to this. Where did you see the other list of values?

Reply:
It's not that important; I was just trying to understand why you did it for one and not the others. Looking at the other cases, however, an array can't or shouldn't be used because they're separate types.